config/segment.yml

```diff
@@ -1,7 +1,7 @@
 base: &default
   enabled: false
   stub: false
-  write_key: 'null'
+  write_key: true
```

Comment:
This is for Segment analytics https://github.com/3scale/porta/blob/99b24fb6622653c0f040e36b5530bd1b55e8f64b/app/views/provider/_analytics.html.erb#L7

We use it in SaaS, on-premises it's disabled. This `true` value was in `openshift` configs, but the value actually doesn't matter, cause as long as it's disabled, it won't be used.

We can actually add env vars here, so we can later reuse this config in SaaS. But not sure whether to leave it for another PR.

Reply:
If it's enabled by default in all environment, how they enable it in SaaS? I guess they need to create a configmap for this file and mount it. I think it would be better to add an env variable to enable it, and leave it disable by default. Anyway, better to do it in another PR I think.